SAGESharp/IO/Binary/TreeBasedSerialization/TreeBuilder.cs

```diff
@@ -144,11 +144,11 @@ private static void ValidateSerializableProperties(Type type, IOrderedEnumerable
             int count = properties.Count();
             if (count == 0)
             {
-                throw new BadTypeException(type, "Type doesn't have serializable properties.");
+                throw new BadTypeException(type, $"Type {type} doesn't have serializable properties.");
             }
             else if (count != properties.Select(p => p.Order).Distinct().Count())
             {
-                throw new BadTypeException(type, "Type has two or more properties with the same serialization order.");
+                throw new BadTypeException(type, $"Type {type} has two or more properties with the same serialization order.");
```

Comment:
The tests are failing, you will need to update the expected exception message in the respective unit tests